thrift/admin.thrift

```diff
@@ -265,6 +265,26 @@ service AdminService {
       2: shared.EntityNotExistsError    entityNotExistError,
       3: shared.InternalServiceError    internalServiceError,
     )
+
+  GetGlobalIsolationGroupsResponse GetGlobalIsolationGroups(1: GetGlobalIsolationGroupsRequest request)
+    throws (
+      1: shared.BadRequestError badRequestError,
```

Comment:
I think we should also include `shared.InternalServiceError`, and I'm not sure if there is any other error which should be included.

Reply:
No, that's always an error. There's no valid scenario where returning an internal-service-error is expected. It is, by definition, an unexpected error. 

Adding it to the list here is to imply that this is expected when it isn't. This, regrettably was an error made through a lot of Uber services earlier and there's been a lot of work to undo.